cli/templates/magento2.php

```diff
@@ -0,0 +1,47 @@
+<html>
```

Comment:
This can be removed

Reply:
Actually it’s not Valet+ specific. It’s used on line 113 of the driver

Reply:
Ah alright 👍 🙌

Reply:
... but is it really needed? A simple `echo` could accomplish the same purpose.

Reply:
My intent was to keep it as close as possible to the downstream version so any future improvements from either side would be easy to merge. Plus I like the display of the page with a clickable link to start the web installer.